flang/runtime/inquiry.cpp

```diff
@@ -15,10 +15,26 @@
 #include "tools.h"
 #include "flang/Runtime/descriptor.h"
 #include <algorithm>
-#include <stdio.h>
 
 namespace Fortran::runtime {
 
+// Helper to store integer value in result[at].
+template <int KIND> struct StoreIntegerAt {
```

Comment:
I see there are similar helpers in in time-intrinsics.cpp. Maybe it would be useful to share the definition in descriptor.h or some other tools.h.

Reply:
Good idea.  I'll do it.